src/workshop.md

```diff
@@ -25,71 +18,101 @@ directions for the project.
 
 ### Registration
 
-Registration closed on October 18th.
+If you plan to attend, [please register no later than **October 18th**](https://forms.gle/voPNCQWwXDHZ96uQ6).
```

Comment:
We should probably ask folks that want to present to have their registration in by an earlier date, maybe Oct1?

Reply:
Yep, we can do that